stackslib/src/chainstate/nakamoto/tenure.rs

```diff
@@ -528,6 +528,17 @@ impl NakamotoChainState {
         }
     }
 
+    /// Get the nakamoto tenure by id
+    pub fn get_nakamoto_tenure_change_by_tenure_id(
```

Comment:
N.B. this doesn't deal with tenure forks.  There could be multiple Nakamoto tenures with the same `tenure_consensus_hash`

Reply:
I don't think that there could be multiple tenures with the same consensus hash. The only permitted forking scenarios would be triggered by block commits (i.e., a tenure's first block is a sibling of another tenure's first block).

Reply:
Yes; that's what I'm referring to.